src/osdc/Objecter.h

```diff
@@ -1213,7 +1592,8 @@ class Objecter : public md_config_obs_t, public Dispatcher {
 public:
   Messenger *messenger;
   MonClient *monc;
-  Finisher *finisher;
+  boost::asio::io_context& ioctx;
+  boost::asio::io_context::strand linger_strand;
```

Comment:
rfe: replace the `ceph::timer<ceph::coarse_mono_clock> timer;` thread with asio timers

Reply:
Will be done in a followup commit.